pennylane/math/single_dispatch.py

```diff
@@ -201,6 +201,7 @@ def _take_autograd(tensor, indices, axis=None):
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "sinc"] = "tensorflow.experimental.numpy"
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "isclose"] = "tensorflow.experimental.numpy"
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "atleast_1d"] = "tensorflow.experimental.numpy"
+ar.autoray._SUBMODULE_ALIASES["tensorflow", "all"] = "tensorflow.experimental.numpy"

```

Comment:
Why was this required?

Reply:
It's because `all` is used in one of the validation checks, but `all` doesn't exist as a top-level function in tensorflow. This line tells autoray to look for it in the `tf.experimental.numpy` submodule.